modules/installation-network-user-infra.adoc

```diff
@@ -118,7 +159,7 @@ This section provides details about the ports that are required.
 ifndef::restricted,origin[]
 [IMPORTANT]
 ====
-In connected {product-title} environments, all nodes are required to have internet access to pull images
+In connected {product-title} environments, all nodes are required to have Internet access to pull images
 for platform containers and provide telemetry data to Red Hat.
 ====
```

Comment:
You'll need to unset the `rhv` values set at the beginning. E.g.,
```
ifeval::["{context}" == "installing-rhv-user-infra"]
:!rhv:
endif::[]

Reply:
OK. Also, right before that, I added:
```
ifeval::["{context}" == "installing-rhv-restricted-network"]
:rhv:
endif::[]
```